benchmarks/bench_latency.py

```
@@ -0,0 +1,374 @@
+        prompt = make_prompt(ctx_len) if ctx_len > 100 else "The meaning of life is"
+
+        for bs in batch_sizes:
+            prompts = [prompt] * bs
+            sp_short = SamplingParams(temperature=0.0, max_tokens=SHORT_TOKENS)
+            sp_long = SamplingParams(temperature=0.0, max_tokens=LONG_TOKENS)
+
+            # Warmup this config
+            try:
+                llm.generate(prompts, sp_short)
+                llm.generate(prompts, sp_long)
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)